factcast-store/src/main/java/org/factcast/store/internal/CondensedQueryExecutor.java

```diff
@@ -122,14 +122,14 @@ public void onEvent(FactInsertionNotification ev) {
   @VisibleForTesting
   boolean mightMatch(String ns, String type) {
     return ns == null
-        ||
         // listens to this exact type
```

Comment:
not really important but it kinda duplicated the comments... 

![image](https://github.com/user-attachments/assets/246ad4f9-ba71-4958-a016-5a223fb23b89)


```suggestion
```